store/schema-update-playbook.md

```diff
@@ -0,0 +1,26 @@
+# Bytebase Schema Update Playbook
+
+All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning. This document goes over the scheme update process during development and release.
+
+## Development
+We use different schema versions for [dev](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go) and [release](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_release.go). 
+
+If version numbers are the same,
+1. Create a new directory under [migration directory](https://github.com/bytebase/bytebase/tree/main/store/migration) with the new version, e.g. 1.3.0.
+2. Copy over the [latest.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest.sql) schema file and [latest_data.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest_data.sql) data filefrom the previous version. And make a PR for changes above.
+3. Bump up the MINOR version in the [dev schema version](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go), e.g. 1.2.0 will be 1.3.0.
+4. Add a new SQL file with DDL statements with a descriptive name, e.g. [sheet_vcs.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/sheet_vcs.sql).
+5. Update latest.sql and latest_data.sql data file w.r.t. DDL changes.
+6. Update the demo data in the test directory.
+
+If version numbers are different,
+1. Continue from step 4 above. Add DDL file and update latest.sql and latest_data.sql.
+The changes in DDL files should be independent from each other. This allows us to move forward with the release independently.
```

Comment:
I don't quite get this. Having a deterministic order to apply DDL changes is a plus. Because it reduces the burden on the developer to develop dependency-free DDL changes. It's also easy for the reasoning since everyone knows how the schema is applied.

Reply:
Shall we have a prefix 001_vcs_sheet.sql then?

Reply:
That's my thought. 001 which means PATCH version 1 (we should start with 000 BTW)